### ISSUE TITLE
[Regression] ones(0,1) + ones(1,0)

### ISSUE THREAD

**Issue by Nelson-numerical-software** [Nelson-numerical-software/nelson#351]
0.4.11
```
ones(0,1) + ones(1,0)
```

```
Error: 
Size mismatch on arguments to arithmetic operator +
```

Expected:
```
[]
```

others cases:
```
ones(0,1) - ones(1,0)
ones(0,1) * ones(1,0)
ones(0,1) .* ones(1,0)
ones(0,1) ./ ones(1,0)
```

**Comment by Nelson-numerical-software** [comment]
see book by Carl de Boor in An Empty Exercise, SIGNUM, Volume 25, pages 2-6, 1990 and C. N. Nett and W. M. Haddad, in A System-Theoretic Appropriate Realization of the Empty Matrix Concept, IEEE Transactions on Automatic Control, Volume 38, Number 5, May 1993

```
v * [](mxn) = [](mxn) * v = [](mxn)

    [](mxn) + [](mxn) = [](mxn)

    [](0xm) *  M(mxn) = [](0xn)

     M(mxn) * [](nx0) = [](mx0)

    [](mx0) * [](0xn) =  0(mxn)
```

add tests and need to fix regression 0.3.10 ....

**Comment by Nelson-numerical-software** [comment]
[Carl de Boor "An empty exercise"](https://github.com/Nelson-numerical-software/nelson/files/5641760/empty.pdf)
